<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AthleteTrackerView</class>
 <widget class="QMainWindow" name="AthleteTrackerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AthleteTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BrowseBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProcessBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="filePathLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Video</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenLogBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open log file</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>631</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
